phonotype.lua

```diff
@@ -289,13 +300,13 @@ function keyboard.code(key, value)
         editing = model:get(editing_script, edit_row)
         edit_col = string.len(editing) + 1
       end
-    elseif key == "C" and keyboard.ctrl() then
+    elseif key == "C" and keyboard.alt() then
```

Comment:
Let's make these `(keyboard.alt() or keyboard.ctrl())` — I just reread the teletype manual, and it responds to both.